plugins/vscode/vscode.plugin.zsh

```diff
@@ -1,30 +1,44 @@
-# VScode zsh plugin
+# VS Code (stable / insiders) / VSCodium zsh plugin
 # Authors:
 #   https://github.com/MarsiBarsi (original author)
 #   https://github.com/babakks
+#   https://github.com/SteelShot
 
-# Use the stable VS Code release, unless the Insiders version is the only
-# available installation
-if ! which code > /dev/null && which code-insiders > /dev/null; then
-  : ${VSCODE:=code-insiders}
-else
-  : ${VSCODE:=code}
-fi
+_vscode-register-aliases () {
+  alias vsc="$VSCODE ."
+  alias vsca="$VSCODE --add"
+  alias vscd="$VSCODE --diff"
+  alias vscg="$VSCODE --goto"
+  alias vscn="$VSCODE --new-window"
+  alias vscr="$VSCODE --reuse-window"
+  alias vscw="$VSCODE --wait"
+  alias vscu="$VSCODE --user-data-dir"
+
+  alias vsced="$VSCODE --extensions-dir"
+  alias vscie="$VSCODE --install-extension"
+  alias vscue="$VSCODE --uninstall-extension"
 
-# Define aliases
-alias vsc="$VSCODE ."
-alias vsca="$VSCODE --add"
-alias vscd="$VSCODE --diff"
-alias vscg="$VSCODE --goto"
-alias vscn="$VSCODE --new-window"
-alias vscr="$VSCODE --reuse-window"
-alias vscw="$VSCODE --wait"
-alias vscu="$VSCODE --user-data-dir"
+  alias vscv="$VSCODE --verbose"
+  alias vscl="$VSCODE --log"
+  alias vscde="$VSCODE --disable-extensions"
+}
 
-alias vsced="$VSCODE --extensions-dir"
-alias vscie="$VSCODE --install-extension"
-alias vscue="$VSCODE --uninstall-extension"
+# Verify if any manual user choice of VS Code exists first.
+# Otherwise, try to detect a flavour of VS Code.
+if ! [ -z ${VSCODE+x} ] && which $VSCODE > /dev/null; then
```

Comment:
Sorry, but I think it's better to let the user see the error messages if the aliases failed to work. Our goal is to make things simple, not undetectable or hard to trace.

What do you think @MarsiBarsi?